src/email/mailgun.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@nestjs/common'
+import { ConfigService } from '@nestjs/config'
+import * as FormData from 'form-data'
+import Mailgun, { MailgunMessageData } from 'mailgun.js'
+import { IMailgunClient } from 'mailgun.js/Interfaces'
+
+const EMAIL_DOMAIN = 'mg.goodparty.org'
+
+export type EmailData = MailgunMessageData & { variables?: Record<string, any> }
+
+@Injectable()
+export class MailgunService {
+  private mailgun: Mailgun
+  private client: IMailgunClient
+
+  constructor(private config: ConfigService) {
+    this.mailgun = new Mailgun(FormData)
+    this.client = this.mailgun.client({
+      key: this.config.get('MAILGUN_API_KEY') as string,
```

Comment:
This is a good example of something we _should_ be able to just pull off of `process.env.MAILGUN_API_KEY`. I don't think we need this value available to anything other than this service, so probably doesn't merit the need for a specific config to this module IMHO.